content/enterprise/before-installing/reference-architecture/vmware.mdx

```diff
@@ -34,10 +34,10 @@ architecture.
 ## Infrastructure Requirements
 
 Depending on the chosen [operational
-mode](/enterprise/before-installing/#operational-mode-decision),
-the infrastructure requirements for Terraform Enterprise range from a single virtual machine
-for demo or proof of concept installations, to multiple virtual machines
-hosting the Terraform Cloud application, PostgreSQL, Redis, and external Vault servers for
+mode](/enterprise/before-installing/#operational-mode-decision), the
+infrastructure requirements for Terraform Enterprise range from a single
+virtual machine for _Mounted Disk_ to multiple virtual machines hosting the
```

Comment:
I see what you did here - setting off the name of the mode in italics. BUT New guidance is in the works to recommend generally not using italics. I think probably we want to say "mode" after to indicate that this is a thing. I'm so sorry - could you possibly go through one more time and update that? The final guidance was just proposed, so I want to get ahead of it.

Also to make this sentence clearer, can we try:

"The operational mode determines the infrastructure requirements for your instance. For example, an installation in Mounted Disk mode may require a single virtual machine, whereas a stateless production installation may require multiple virtual machines to host the Terraform Cloud application, PostgreSQL, Redis, and external Vault servers."